### PR TITLE
Support Tabstrip  `activeTabIndex` to be null

### LINKED ISSUE #1676: Tabstrip throws error when passing `null` to `activeTabIndex`
### Package name(s)

Lab v1.0.0-alpha.4 (@salt-ds/lab)

### Description

I want to find a way to not set any active tab, which i understand is not a standard use case. However, when attempting to pass `null` to `Tabstrip` component (following similar convention to `List` for example), there is error being thrown.

### Steps to reproduce

```
<Tabstrip activeTabIndex={null}>
```

Stackblitz reproduce: https://stackblitz.com/edit/salt-template-feca2y?file=App.tsx

### Expected behavior

The code should not throw any error.

Even better, showing no tab being active state.

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

### PR DESCRIPTION
**Pull request by origami-z**
Fixes #1676

### CODE REVIEW

File: packages/lab/stories/tabstrip/tabstrip.stories.tsx [hunk 1]
```diff
@@ -1039,3 +1039,23 @@ export const DraggableTabsWithOverflow = () => {
     </div>
   );
 };
+
+export const NullTabIndex = () => {
```
Comment (origami-z) [inline packages/lab/stories/tabstrip/tabstrip.stories.tsx:1043]: Use this story to test

**Comment by origami-z** [comment]
Tested by MCP team, to address clicking active tab not firing 'fake' inactive tab before having the PR change

**Review by joshwooding — COMMENTED** [review]
I'm not too sure about this... I understand that some teams might want to do this but we've previously spoken about this being a hard requirement of tabs

**Comment by origami-z** [comment]
> I'm not too sure about this... I understand that some teams might want to do this but we've previously spoken about this being a hard requirement of tabs

We have to do something for MCP right now, given no other option will be available.

Either we officially support in TS, or something like #1683

File: .changeset/brown-sloths-change.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
+---
+
+Fix Tabstrip crashes when setting `activeTabIndex` to null
```
Comment (joshwooding) [inline .changeset/brown-sloths-change.md:5]:
```suggestion
Fix Tabstrip crashing when `activeTabIndex` is set to null
```

**Comment by ZarkoPernar** [comment]
> I'm not too sure about this... I understand that some teams might want to do this but we've previously spoken about this being a hard requirement of tabs

Why a hard requirement? It will obviously be used with no selection in some scenarios.

**Comment by joshwooding** [comment]
> > I'm not too sure about this... I understand that some teams might want to do this but we've previously spoken about this being a hard requirement of tabs
> 
> Why a hard requirement? It will obviously be used with no selection in some scenarios.

Technically we don't support those scenarios. A requirement for Tabs is that it always has an active item. I think it's worth merging this anyway and relying on documentation to convey this.

**Comment by origami-z** [comment]
> Why a hard requirement? It will obviously be used with no selection in some scenarios.

When Homepage (being not a part of tab) is active, none of the view tabs should be active. If they override css pure visually, then `onActiveChange` won't fire when clicking back.

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**